SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/OwnershipLockingPolicy.cpp

```diff
@@ -11,7 +11,7 @@
 
 DEFINE_LOG_CATEGORY(LogOwnershipLockingPolicy);
 
-bool UOwnershipLockingPolicy::CanAcquireLock(const AActor* Actor) const
```

Comment:
This still looks nice and const, what's the deal?

Reply:
static now bruh

Reply:
Ah gotcha. Why though, just because we can?

Reply:
This seems like the kind of thing someone is going to want to make virtual at some point. Although for the same reason we can make it static now it's probably not that big a deal to make it not static later.